src/components/Inputs/InputMonthAndYear.jsx

```diff
@@ -7,25 +7,25 @@ import { useStyles } from './InputStyle';
 export const InputMonthAndYear = ({ label, value, setValue }) => {
   const classes = useStyles();
 
-//   const handleChange = ({ target: { value }}) => setValue && setValue(value);
+  //   const handleChange = ({ target: { value }}) => setValue && setValue(value);
 
   const formats = {
-    monthAndYear: "MM/yyyy",
+    monthAndYear: 'MM/yyyy',
   };
 
   return (
     <LocalizationProvider dateAdapter={AdapterDateFns} dateFormats={formats}>
       <DatePicker
         views={['year', 'month']}
-      
         label={<div className={classes.lbl2}>{label}</div>}
         minDate={new Date('1990-01-01')}
         maxDate={Date.now()}
         value={value}
-        onChange={(value)=> setValue && setValue(value)}
-//           onChange={handleChange}
+        onChange={(value) => setValue && setValue(value)}
+        //           onChange={handleChange}
         renderInput={(params) => (
           <TextField
+            required={true}
```

Comment:
Only required is enough
For Boolean when you pass only props name it will be interpreted as true